api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/x509/X509Properties.java

```diff
@@ -287,6 +296,21 @@
      */
     private String name;
 
+    /**
+     * Whether to extract certificate from request via ExtractSSLCertificate interface.
```

Comment:
Remove the reference to the interface; Only add if you can actually link to the component.